File: src/common/1.21.1/main/java/settingdust/mod_sets/v1_21/util/MinecraftAdapter.java
package settingdust.mod_sets.v1_21.util;

import net.minecraft.resources.ResourceLocation;

public class MinecraftAdapter implements settingdust.mod_sets.util.MinecraftAdapter {
    @Override
    public ResourceLocation id(final String namespace, final String path) {
        return ResourceLocation.fromNamespaceAndPath(namespace, path);
    }
}

File: src/common/1.21.1/main/kotlin/settingdust/mod_sets/v1_21/util/MinecraftAdapter.kt
package settingdust.mod_sets.v1_21.util

import net.minecraft.resources.ResourceLocation
import settingdust.mod_sets.util.MinecraftAdapter

class MinecraftAdapter : MinecraftAdapter {
    override fun id(namespace: String, path: String) = ResourceLocation.fromNamespaceAndPath(namespace, path)
}